assets/app/views/_templateopt.html

```diff
@@ -0,0 +1,21 @@
+<div class="template-options" ng-show="hasParameters">
```

Comment:
can we make this an ng-if instead so the DOM doesn't even get rendered when it doesn't need to be